frontend/tests/util/blocks.ts

```diff
@@ -0,0 +1,238 @@
+                    await saveButton.click();
+
+                    await expect(
+                        page.getByRole("heading", { name: newSeries.title })
+                    ).not.toBeVisible();
+                    await expect(
+                        page.getByText("Cat software running on dog hardware.")
+                    ).not.toBeVisible();
+                });
+
+                await test.step("Description and videos", async () => {
+                    await editButton.click();
+                    await page.getByLabel("Show description").setChecked(true);
+                    await saveButton.click();
+
+                    await expect(
+                        page.getByRole("heading", { name: newSeries.description })
+                    ).not.toBeVisible();
+                    await expect(page.getByText(newSeries.description).first()).toBeVisible();
+                });
+
+                await test.step("Title, description and videos", async () => {
+                    await editButton.click();
+                    await page.getByLabel("Show title").setChecked(true);
+                    await page.getByLabel("Show description").setChecked(true);
+                    await saveButton.click();
+
+                    await expect(
+                        page.getByRole("heading", { name: newSeries.title })
+                    ).toBeVisible();
+                    await expect(page.getByText(newSeries.description).first()).toBeVisible();
+                });
+            });
+            break;
+        }
+        case "video": {
+            await test.step("Video can be changed", async () => {
+                const input = page.getByRole("combobox");
+                await input.pressSequentially("Dual Stream Cats");
+                await page.getByRole("option", { name: "Dual Stream Cats"}).click();
+
+                await page.keyboard.press("Enter");
+
+                await expect(page
+                    .getByRole("heading", { name: "Video Of A Tabby Cat" })
+                    .first())
+                    .toBeVisible();
+            });
+
+            await test.step("Title can be hidden", async () => {
+                await editButton.click();
+                await page.getByText("Show title").click();
+                await saveButton.click();
+
+                await expect(page.getByRole("heading")).toHaveCount(0);
+            });
+            break;
+        }
+        // case "text": {
+        //     await test.step("Text can be changed", async () => {
+        //         await editText(page, "The lazy dog jumps over the quick brown fox.");
+        //     });
+        //     break;
+        // }
+        // case "title": {
+        //     await test.step("Title can be changed", async () => {
+        //         await editText(page, "Super unique title");
+        //     });
+        //     break;
+        // }
```

Comment:
This one commit contains the commented out version of this. Maybe you can edit the commits a bit so that this in-between version is not in here.